### PR TITLE
Implement "@Redfish.CollectionCapabilities" in the Systems collection

### LINKED ISSUE #271: Implement "@Redfish.CollectionCapabilities" in the Systems collection
It should list all capabilities objects (for all server profile templates)

### PR DESCRIPTION
**Pull request by victorhugorodrigues**
Closes #271

### CODE REVIEW

File: oneview_redfish_toolkit/api/computer_system_collection.py [hunk 1]
```diff
@@ -14,8 +14,14 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+from collections import OrderedDict
```
[the remark this answers is not in the document]
Comment (victorhugorodrigues) [reply oneview_redfish_toolkit/api/computer_system_collection.py]: Done.

File: oneview_redfish_toolkit/api/computer_system_collection.py [hunk 2]
```diff
@@ -28,7 +34,7 @@ class ComputerSystemCollection(RedfishJsonValidator):
 
     SCHEMA_NAME = 'ComputerSystemCollection'
 
-    def __init__(self, server_hardware_list):
+    def __init__(self, server_hardware_list, server_profile_templates):
```
[the remark this answers is not in the document]
Comment (victorhugorodrigues) [reply oneview_redfish_toolkit/api/computer_system_collection.py]: Done.

File: oneview_redfish_toolkit/tests/api/test_computer_system_collection.py [hunk 3]
```diff
@@ -45,12 +45,19 @@ def setUp(self):
         ) as f:
```
Comment (ricardogpsf) [inline oneview_redfish_toolkit/tests/api/test_computer_system_collection.py]: Update Copyright to 2017-2018.

File: oneview_redfish_toolkit/blueprints/computer_system_collection.py [hunk 4]
```diff
@@ -39,8 +39,13 @@ def get_computer_system_collection():
     # Gets all server hardware
```
Comment (ricardogpsf) [inline oneview_redfish_toolkit/blueprints/computer_system_collection.py]: Update Copyright to 2017-2018.

File: oneview_redfish_toolkit/tests/api/test_computer_system_collection.py (hunk 3 above)
Comment (victorhugorodrigues) [reply oneview_redfish_toolkit/tests/api/test_computer_system_collection.py]: Done.

File: oneview_redfish_toolkit/blueprints/computer_system_collection.py (hunk 4 above)
Comment (victorhugorodrigues) [reply oneview_redfish_toolkit/blueprints/computer_system_collection.py]: Done.

**Pull request merged**